### DISCUSSION TITLE
使用fastquit模组会导致报错，但是没有问题出现

### DISCUSSION THREAD

**Discussion by iPodtouch6557** [Meloong-Git/PCL#4128] [提问]
如题，每次我玩都得来这么一下，很烦……（因为不确定是不是PCL的问题现在这里问下）
[错误报告-2024-6-30_22.09.39.zip](https://github.com/user-attachments/files/16044975/-2024-6-30_22.09.39.zip)

**Comment by wuliaodexiaoluo** [comment]
![f937c8b0be1ce31b2adae231ee4090a8](https://github.com/Hex-Dragon/PCL2/assets/160083863/19bba608-9ea2-43d6-9db4-b0e8101d0d7a)

应该是 Mod 冲突，选一个删掉

也可以具体描述下是怎么个报错法，崩溃还是只是显示不兼容

日志还有几个 IO 异常，不知道会不会影响到游戏体验

**Reply by iPodtouch6557** [reply]
崩溃，未知原因
其他异常不影响
